src/swish/json.ss

```diff
@@ -620,15 +620,26 @@
         (let-values ([(op get) (open-string-output-port)])
           (write-string (json-key->string k) op)
           (get)))
+      (define (format-value wr val)
+        (and (free-identifier=? wr #'json:write)
+             (or (boolean? val) (fixnum? val) (string? val))
```

Comment:
Do you want to include `(and (flonum? val) (finite? val))`?

Reply:
Do you want to include `(eqv? val #\nul)`?

Reply:
We may as well. I pushed an update to do this (stopping short of bignums) and added some tests.